go/vt/vtgate/safe_session.go

```diff
@@ -572,6 +572,28 @@ func (session *SafeSession) TimeZone() *time.Location {
 	return loc
 }
 
+// ForeignKeyChecks returns the foreign_key_checks stored in system_variables map in the session.
+func (session *SafeSession) ForeignKeyChecks() (*bool, *string) {
```

Comment:
This function is only used in one place and we always ignore the second returned argument. We can just remove it.